subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junit/JUnitTestEventAdapter.java

```diff
@@ -29,20 +29,27 @@
 import org.junit.runner.notification.RunListener;
 
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.Map;
-import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class JUnitTestEventAdapter extends RunListener {
+    private class JUnitTestEventInternalDescriptor {
+        public TestDescriptorInternal descriptor;
```

Comment:
Fields should never be public.

Reply:
Didn't think this was critical for private nested classes, but I'll add boilerplate accessors and setters